examples/super_resolution/android/app/src/main/java/com/google/aiedge/examples/super_resolution/MainActivity.kt

```diff
@@ -58,6 +58,7 @@ import androidx.compose.ui.unit.sp
 import com.google.aiedge.examples.super_resolution.gallery.ImagePickerScreen
 import com.google.aiedge.examples.super_resolution.quicksample.QuickSampleScreen
```

Comment:
Just noticed that you have these called "quicksample". I'm guessing that's a mistake?